### PR TITLE
Backport multiple pre-checks in 2.7

### PR DESCRIPTION
**Pull request by gdemonet**
Backports of:
- #3050
- #3065
- #3069
- #3076
- #3107

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +45,7 @@
   (PR [#3120](https://github.com/scality/metalk8s/pull/3120))
 
 ## Release 2.7.1
+
```
Comment (TeddyAndrieux) [inline CHANGELOG.md]: A space were added here and not for `2.7.2` ? :smiley:

**Review by TeddyAndrieux — DISMISSED** [review]
You want to backport this one as well otherwise bootstrap restore is broken #3107

File: CHANGELOG.md (hunk 1 above)
Comment (gdemonet) [reply CHANGELOG.md]: Removed.

**Comment by TeddyAndrieux** [comment]
LGTM, just shouldn't we remove entry from 2.8.0 changelog since this one is not released yet ?

**Comment by gdemonet** [comment]
> LGTM, just shouldn't we remove entry from 2.8.0 changelog since this one is not released yet ?

Oh good point, right, will do this change.

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**